projects/cdk/schematics/ng-update/v4/index.ts

```diff
@@ -44,6 +44,7 @@ function main(options: TuiSchema): Rule {
         replaceIdentifiers(options, IDENTIFIERS_TO_REPLACE);
         removeModules(options, MODULES_TO_REMOVE);
         renameTypes(options, TYPES_TO_RENAME);
+        saveActiveProject();
```

Comment:
why?

Reply:
Without this line the test will not work, you can check it yourself

https://github.com/taiga-family/taiga-ui/pull/7967/files#diff-915dfc400271ffd22453275dc5527158f559fd5fee119d18f48290d77336595fR37

Reply:
@vladimirpotekhin ping